pkgs/by-name/pi/picocrypt-cli/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "picocrypt-cli";
+  version = "2.04";
+
+  src = fetchFromGitHub {
+    owner = "Picocrypt";
+    repo = "CLI";
+    rev = version;
+    hash = "sha256-LWU/u+5AJgBS9IAA3fq5jQLlst+kpEqtRZvjkBSMG2E=";
+  };
+
+  sourceRoot = "${src.name}/picocrypt";
+  vendorHash = "sha256-5bdiXy/c4FYu544lyL/ac3KF/1zWIPG/bmrpdZa9YV8=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  CGO_ENABLED = 1;
+
+  meta = {
+    description = "A command-line interface for Picocrypt";
+    homepage = "https://github.com/Picocrypt/CLI";
+    license = lib.licenses.gpl3Only;
+    maintainers = with lib.maintainers; [ arthsmn ];
+    mainProgram = "picocrypt";
+    platforms = lib.platforms.all;
```

Comment:
```suggestion
```
Default is fine here https://github.com/NixOS/nixpkgs/blob/084fddb13f1275c630361b7d2eae5a79f8478504/pkgs/build-support/go/module.nix#L314

Reply:
Done.